fix: video download fails when applying watermark

### Summary

Downloading some videos fails during watermark application.

### Environment

iOS

### Steps to Reproduce

1. Open a video.
2. Start the download flow.
3. Apply a watermark.
4. Complete the download.

### Actual Result

The download fails for some videos when the watermark is applied.

### Expected Result

Videos should download successfully with the watermark applied.

### Regression?

Unknown

### Evidence

This was reported on discord [here](https://discord.com/channels/1470168817589158040/1470255950408454164/1493984612228141076).

Logs:
```
[2026-04-15T07:38:50.916514] [DEBUG] [WatermarkDownloadService] video: Using cached video file
[2026-04-15T07:38:50.931508] [DEBUG] [PooledPlayer] video: play_started index=1 videoId=4f91df165b76284878578771561fb45e0e4d371fd4ec545b5e9f9053adfdd54d url=https://media.divine.video/e6c735f13e845a946095cca1e6fe4ff58a5a12e81e8f3e8ce33020b08d29d298/720p.mp4 issuedPlay=true playing=true elapsedMs=null
[2026-04-15T07:38:50.933459] [DEBUG] [FollowRepository] system: Indexer wss://purplepag.es returned 34 followers for 5ab67f7d7fed4f781008c0ec0d26c8113f9fb46094a8346246c70c75e75db9fb
[2026-04-15T07:38:50.933498] [INFO] [FollowRepository] system: Indexer followers counts: [34, 15, 90], using 90 for 5ab67f7d7fed4f781008c0ec0d26c8113f9fb46094a8346246c70c75e75db9fb
[2026-04-15T07:38:50.933760] [DEBUG] [FollowRepository] system: Follower stats fetched: FollowerStats(followers: 91, following: 28)
[2026-04-15T07:38:50.937525] [DEBUG] [WatermarkDownloadService] video: Video dimensions from file: 720x1280
[2026-04-15T07:38:51.049056] [ERROR] [WatermarkDownloadService] video: Failed to render watermarked video: PlatformException(RENDER_ERROR, No source tracks available for compositing (sourceTrackIDs: 0, configTrackID: 1), null, null)
```

--- a/mobile/pubspec.yaml
+++ b/mobile/pubspec.yaml
@@ -273,7 +273,7 @@ dependencies:
   firebase_messaging: ^16.1.1
   convert: ^3.1.2
   audio_session: ^0.2.2
-  pro_video_editor: ^1.15.0
+  pro_video_editor: ^1.16.1
   pro_image_editor: ^12.4.4
   flutter_colorpicker: ^1.1.0
   file_picker: ^10.3.7

